ca/ca.go

```diff
@@ -143,16 +149,33 @@ func (ca *CA) Init(config *authority.Config) (*CA, error) {
 		acmeRouterHandler.Route(r)
 	})
 
-	/*
-		// helpful routine for logging all routes //
-		walkFunc := func(method string, route string, handler http.Handler, middlewares ...func(http.Handler) http.Handler) error {
-			fmt.Printf("%s %s\n", method, route)
-			return nil
-		}
-		if err := chi.Walk(mux, walkFunc); err != nil {
-			fmt.Printf("Logging err: %s\n", err.Error())
-		}
-	*/
+	scepPrefix := "scep"
+	scepAuthority, err := scep.New(auth, scep.AuthorityOptions{
+		Service: auth.GetSCEPService(),
+		DB:      auth.GetDatabase().(scep.DB),
+		DNS:     dns,
+		Prefix:  scepPrefix,
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "error creating SCEP authority")
+	}
+	scepRouterHandler := scepAPI.New(scepAuthority)
+	mux.Route("/"+scepPrefix, func(r chi.Router) {
+		scepRouterHandler.Route(r)
+	})
```

Comment:
I've just seen that scep.DB only has `StoreCertificate(crt *x509.Certificate) error`, there's no need to "cast" nor check that `db.AuthDB` from `GetDatabase()` implements it, because StoreCertificate is a method in db.AuthDB. 

But see my comment below, I don't think you need to use StoreCertificate.